packages/lib/createAProfileForAnExistingUser.ts

```diff
@@ -25,6 +25,16 @@ export const createAProfileForAnExistingUser = async ({
   if (!org) {
     throw new Error(`Organization with id ${organizationId} not found`);
   }
+
+  const existingProfile = await ProfileRepository.findByUserIdAndOrgId({
```

Comment:
Purpose of the fn is to create a profile which already exists so we can safely return early.

Reply:
Makes sense! Nice job - didnt think of this